basic_system/src/system_implementation/system/io_subsystem.rs

```diff
@@ -298,7 +298,7 @@ where
                     Bytes32::ZERO
                 } else {
                     // EOA case:
-                    Bytes32::from_u256_be(U256::from_limbs([
+                    Bytes32::from_u256_be(&U256::from_limbs([
```

Comment:
This would be better to be outlined constant, but can do it separately (I also had const hex-init for Bytes32 stashed somewhere)